lib/clone_kit/graph.rb

```diff
@@ -26,7 +26,7 @@ def tsort_each_node(&block)
     end
 
     def tsort_each_child(node, &block)
-      @vertices[node].each(&block)
+      @vertices[node]&.each(&block)
```

Comment:
🤔 Under what circumstances do we run into an issue now that we didn't before?  Or was this always a potential issue?